R/truetype.r

```diff
@@ -150,7 +150,13 @@ ttf_find_default_path <- function() {
     return(paths[file.exists(paths)])
 
   } else if (grepl("^mingw", R.version$os)) {
-    return(paste(Sys.getenv("SystemRoot"), "\\Fonts", sep=""))
+    paths <-
+      c(paste(Sys.getenv("SystemRoot"), "\\Fonts", sep=""),
```

Comment:
While you're in here, can you change this to `file.path(Sys.getenv("SystemRoot"), "Fonts")` instead of `paste()`?

Reply:
sure! will upload those soon